src/System.Runtime/tests/NSStatus/String.NSStatus.cs

```diff
@@ -0,0 +1,811 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This test is machine generated. Please see the readme.md next to the test for more information. Please do not modify this test by hand.
+
+using System;
+using Xunit;
+public class NSStatus
+{
+    [Fact]
+    public static void StringTest()
+    {
+        bool temp;
+        int errcount = 0;
+        //*** System.String.Split-0x801fbe4aea4c2406-101 ***
+        temp = false;
+        // erid(0x8c85191e291f5266
+        object Param_0_005d850a = (System.String)("SDK, desktop, application, exe, dll");
+        string[] Local_0_1 = ((System.String)Param_0_005d850a).Split(new char[] {';'},(System.StringSplitOptions)1);
+        string[] Expected_0 = new string[] {"SDK, desktop, application, exe, dll"};
+        temp = (Expected_0.Length == Local_0_1.Length); for(int i=0;i<Expected_0.Length;i++){if(Expected_0[i] != Local_0_1[i]){temp = false; break;}}
+        if(!temp) { Console.WriteLine(@"FAILED -> 0 "); errcount++;}
+        // //Summary 1-classes 1-modules 
```

Comment:
It says that this chunk of the test requires one type from one assembly. More complex tests will have different numbers.